### PR TITLE
Add App Management doc

### PR DESCRIPTION
**Pull request by EugeniyBurmistrov**

### CODE REVIEW

File: docs/cloud-app-management.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+```bash
+$ resolve-cloud list
+ID                             NAME             VERSION       USERID               UPDATE
+rqa6rxkghu8f9f3kjjd2tmak51     hacker-news      0.0.0         myemail@example.com  -> 0.11.0
+z6ze1aqfuedw54qa0gp58xu6ov     shopping-list    0.0.0         myemail@example.com  -> 0.11.0
+f3vck8svysxfx0wu9umug2vefiv0   my-app           0.10.0        myemail@example.com  -> 0.11.0
+```
+
+This output also provides information on what applications can be upgraded to a newer runtime. To upgrade an application's runtime to the latest minor within the current major version, type:
```
Comment (arminal) [inline docs/cloud-app-management.md]: latest minor what? version? make sure you are clear about what you are saying

File: docs/cloud-app-management.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+```bash
+$ resolve-cloud upgrade <app-id> --runtime <runtime-version>
+```
+
+The runtime version should be specified in the SemVer format.
+
+## Enable Logging
+
+For logging to work, make sure to enable the correct debug levels for your application. See the [Debugging](debugging.md) topic for more information.
```
Comment (arminal) [inline docs/cloud-app-management.md]: for logging to work?

File: docs/cloud-app-management.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+$ resolve-cloud tracing enable <deployment>
+```
+
+To view the list of all generated traces:
+
+```
+$ resolve-cloud tracing summary <deployment>
+```
+
+To view details of particular trace:
```
Comment (arminal) [inline docs/cloud-app-management.md]:
```suggestion
To view the details of a particular trace:
```

File: docs/cloud-app-management.md (hunk 1 above)
Comment (EugeniyBurmistrov) [reply docs/cloud-app-management.md]: Added "version"

File: docs/cloud-app-management.md (hunk 2 above)
Comment (EugeniyBurmistrov) [reply docs/cloud-app-management.md]: I removed the "For logging to work" part. Instead, I added a sentence that describes what a debug level is:
"ReSolve groups debug messages into debug levels based on its urgency. Make sure to enable the correct debug levels for your application. See the [Debugging](debugging.md) topic for more information."

**Pull request merged**